connectors/aws/aws-textract/element-templates/aws-textract-outbound-connector.json

```diff
@@ -39,24 +39,6 @@
       "type" : "zeebe:taskDefinition"
     },
     "type" : "Hidden"
-  }, {
```

Comment:
Looks good, thanks!

Two small things:
- Please format the PR description in the standard way ([example](https://github.com/camunda/connectors/pull/3161)). This helps us review faster 😄 
- Don't forget to add this new version to web-modeler (if you weren't planning to already) 😄 

Feel free to resolve this comment yourself!

Reply:
1) changed
2)already have a task for it https://github.com/orgs/camunda/projects/23/views/5?pane=issue&itemId=76168970